sources/assets/Stride.Core.Assets/Compiler/PackageCompiler.cs

```diff
@@ -20,11 +20,9 @@ public class PackageCompiler : IPackageCompiler
         static PackageCompiler()
         {
             // Compute StrideSdkDir from this assembly
-            // TODO Move this code to a reusable method
             var codeBase = typeof(PackageCompiler).Assembly.Location;
-            var uri = new UriBuilder(codeBase);
-            var path = Path.GetDirectoryName(Uri.UnescapeDataString(uri.Path));
-            SdkDirectory = Path.GetFullPath(Path.Combine(path, @"..\.."));
+            // from ../bin/Debug/net{version} -> ../bin
+            SdkDirectory = Directory.GetParent(codeBase)?.Parent?.Parent?.FullName ?? "";
```

Comment:
`Directory` goes through and reads from the filesystem afair while `Path` just works on the string representation of the path. 
You can chain `Path.GetDirectoryName` calls if you want to do something similar, see examples in 
https://learn.microsoft.com/en-us/dotnet/api/system.io.path.getdirectoryname?view=net-8.0#system-io-path-getdirectoryname(system-string)

Reply:
@Eideren Good point. I did a simple benchmark and my fix was about 2 times slower. I started all over and I noticed UriBuilder does nothing so I removed it so it wouldn't throw an exception anymore

Reply:
@Eideren  I get it now why UriBuilder was in the first place! Thats because .NET Framework `Assembly.CodeBase` was returning URI path : https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=net-8.0 which is useless now.